TCAT/Controllers/HomeViewController.swift

```diff
@@ -445,43 +445,41 @@ class HomeViewController: UIViewController, UITableViewDelegate, UITableViewData
         
         if status == .denied {
             
-            var actions: [UIAlertAction] = []
-            
-            let theTitle = "Location Services Disabled"
-            let message = "The app won't be able to use your current location without permission. Tap Settings to turn on Location Services."
-            var leftButton = "Cancel"
-            
-            if let showReminder = userDefaults.value(forKey: Constants.UserDefaults.locationAuthReminder) as? Bool {
-                if showReminder {
-                    actions.append(UIAlertAction(title: "Dismiss", style: .default))
-                    leftButton = "Don't Remind Me Again"
-                } else {
-                    return
-                }
-            } else {
-                userDefaults.set(true, forKey: Constants.UserDefaults.locationAuthReminder)
+            let alertController = UIAlertController(title: "Location Services Disabled", message: "The app won't be able to use your current location without permission. Tap Settings to turn on Location Services.", preferredStyle: .alert)
+            let settingsAction = UIAlertAction(title: "Settings", style: .default) { (_) in
+                UIApplication.shared.open(URL(string: "App-prefs:root=LOCATION_SERVICES") ?? URL(string: UIApplicationOpenSettingsURLString)!, options: [:], completionHandler: nil)
```

Comment:
I didn't know you could do this!

Reply:
yah i think it only works in iOS 11 and Swift 4 (haven't tested it for the older ones) so kept the other one cause that works no matter what